sdk/include/aditof/camera_definitions.h

```diff
@@ -30,6 +31,33 @@ enum class ConnectionType {
     LOCAL     //!< The device is embedded in the system
 };
 
+/**
+ * @struct IntrinsicParameters
+ * @brief Provides the intrinsic parameters of a camera.
+ */
+struct IntrinsicParameters {
+	/**
+	 * @brief The intrinsic 3x3 parameter matrix (K) with values specified in pixel units.
+	 */
+	std::vector<float> cameraMatrix;
+
+	/**
+	 * @brief The distortion coefficients
+	 */
+	std::vector<float> distCoeffs;
+
+	/**
+	 * @brief The width of a sensor unit cell specified in mm.
+	 */
+	float pixelWidth;
+
+	/**
+	 * @brief The height of a sensor unit cell specified in mm.
+	 */
+	float pixelHeight;
+
```

Comment:
Please remove extra line

Reply:
solved in commit https://github.com/analogdevicesinc/aditof_sdk/pull/203/commits/adba4901642e22b8626a1910e3c23271c2af6dc3